Install from source not working

The install from source is not working on the staging branch, giving ClassNotFoundException

```
(serve) ubuntu@ip-172-31-15-18:~/stage/serve$ pip install -U -e .
Obtaining file:///home/ubuntu/stage/serve
Requirement already satisfied, skipping upgrade: Pillow in /home/ubuntu/anaconda3/lib/python3.6/site-packages (from torchserve==0.0.1b20200507) (5.4.1)
Requirement already satisfied, skipping upgrade: psutil in /home/ubuntu/anaconda3/lib/python3.6/site-packages (from torchserve==0.0.1b20200507) (5.4.5)
Requirement already satisfied, skipping upgrade: future in /home/ubuntu/anaconda3/lib/python3.6/site-packages (from torchserve==0.0.1b20200507) (0.18.2)
Installing collected packages: torchserve
 Found existing installation: torchserve 0.0.1b20200507
  Uninstalling torchserve-0.0.1b20200507:
   Successfully uninstalled torchserve-0.0.1b20200507
 Running setup.py develop for torchserve
Successfully installed torchserve
(serve) ubuntu@ip-172-31-15-18:~/stage/serve$ cd ~/work
(serve) ubuntu@ip-172-31-15-18:~/work$ torchserve --ts-config config.properties
Removing orphan pid file.
(serve) ubuntu@ip-172-31-15-18:~/work$ Error: Could not find or load main class org.pytorch.serve.ModelServer
Caused by: java.lang.ClassNotFoundException: org.pytorch.serve.ModelServer
```

--- a/scripts/install_from_src_macos
+++ b/scripts/install_from_src_macos
@@ -31,12 +31,12 @@ install_torch_deps()
 
 build_install_server()
 {
-	cd ../ && pip install .
+	pip install .
 }
 
 build_install_archiver()
 {
-	cd ../model-archiver && pip install .
+	cd model-archiver && pip install .
 }
 
 clean_slate

